deploy/genesis/Dockerfile.testrunner

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
Id love to see this file move into say `./test/intra/Dockerfile.testrunner` or the like

Reply:
Moved it to test/integration with the other files